server/src/test/java/com/objectcomputing/checkins/services/role/RoleControllerTest.java

```diff
@@ -2,6 +2,10 @@
 
```

Comment:
I replaced RoleControllerTest with testcontainer. I could be talked in to keeping the old mockito code, and renaming this.

Reply:
@Bagurp @bhatiam12 @meyerinj I'd like to hear your input on this...I'm also on the fence about it. If I thought we would be able to better protect our quality by having both, I would get behind it. I'm not sure that I can see an argument for us being unable to test everything in the controller with these testcontainer / integration-style tests.

Reply:
@mkimberlin @chrismckiernan tbh I am very new to test containers and this will be a learning for me. that said if we can test everything in controllers using the testcontainer approach then I am ok with getting rid of mockito way.